docs/core/linux-prerequisites.md

```diff
@@ -127,244 +155,417 @@ scl enable rh-dotnet20 bash
 
+3. Download the .NET Core SDK binary  (tarball).
+
 **Fedora 24**
 
 ```bash
 curl -sSL -o dotnet.tar.gz https://go.microsoft.com/fwlink/?linkid=848833
 ```
 
+4. Extract the .NET Core SDK binaries.
+
+```bash
+sudo mkdir -p /opt/dotnet && sudo tar zxf dotnet.tar.gz -C /opt/dotnet
+```
+
+5. Add dotnet to your PATH.
+
+```bash
+sudo ln -s /opt/dotnet/dotnet /usr/local/bin
+```
+
 ---
-3. Extract the .NET Core SDK binaries.
-    ```bash
-    sudo mkdir -p /opt/dotnet && sudo tar zxf dotnet.tar.gz -C /opt/dotnet
-    ```
-4. Add dotnet to your PATH.
-    ```bash
-    sudo ln -s /opt/dotnet/dotnet /usr/local/bin
-    ```
-5. Run the `dotnet --help` command to prove the installation succeeded.
+
+6. Run the `dotnet --version` command to prove the installation succeeded.
 
      ```bash
-     dotnet --help
+     dotnet --version
      ```
-## Install .NET Core for CentOS 7.1 (64 bit) & Oracle Linux 7.1 (64 bit)
 
-> [!Warning]
-> Before you start, remove any previous versions of .NET Core from your system.
+## Install .NET Core for CentOS 7.1 (64 bit) & Oracle Linux 7.1 (64 bit)
 
 To Install .NET Core for CentOS 7.1 (64 bit) & Oracle Linux 7.1 (64 bit):
 
-1. Get the prerequisites.
+1. Remove any **previous preview** versions of .NET Core from your system.
+
+2. Get the prerequisites.
+
     ```bash
-    sudo dnf install libunwind libicu
+    sudo yum install libunwind libicu
     ```
-2. Download the .NET Core SDK binary (tarball).
+
+> [!Note]
+> A user-controlled directory is required for Linux system installs from tar.gz.
 
 # [.NET Core 2.x](#tab/netcore2x)
 
+3. Download the .NET Core SDK binary (tarball).
+
 ```bash
 curl -sSL -o dotnet.tar.gz https://aka.ms/dotnet-sdk-2.0.0-linux-x64
 ```
 
+4. Extract the .NET Core SDK binaries.
+
+```bash
+mkdir -p ~/dotnet && tar zxf dotnet.tar.gz -C ~/dotnet
+```
+
+5. Add dotnet to your PATH.
+
+```bash
+export PATH=$PATH:$HOME/dotnet
+```
+
 # [.NET Core 1.x](#tab/netcore1x)
 
+3. Download the .NET Core SDK binary (tarball).
+
 ```bash
 curl -sSL -o dotnet.tar.gz https://go.microsoft.com/fwlink/?linkid=848821
 ```
 
+4. Extract the .NET Core SDK binaries.
+
+```bash
+sudo mkdir -p /opt/dotnet && sudo tar zxf dotnet.tar.gz -C /opt/dotnet
+```
+
+5. Add dotnet to your PATH.
+
+```bash
+sudo ln -s /opt/dotnet/dotnet /usr/local/bin
+```
+
 ---
-3. Extract the .NET Core SDK binaries.
+
+6. Run the `dotnet --version` command to prove the installation succeeded.
+
+     ```bash
+     dotnet --version
+     ```
+
+## Install .NET Core for SUSE Linux Enterprise Server (64 bit)
+
+To Install .NET Core 2.x for SUSE Linux Enterprise Server (SLES) 12 SP2 (64 bit):
+
+1. Remove any **previous preview** versions of .NET Core from your system.
+
+2. Get the prerequisites.
+
     ```bash
-    sudo mkdir -p /opt/dotnet && sudo tar zxf dotnet.tar.gz -C /opt/dotnet
+    sudo zypper install libunwind libicu
     ```
-4. Add dotnet to your PATH.
+
+3. Download the .NET Core SDK binary (tarball).
+
+```bash
```

Comment:
this indentation is different than others:

![image](https://user-images.githubusercontent.com/12971179/29892532-bec36078-8d83-11e7-886b-204885ebc1b7.png)